### PR TITLE
add opening brackets

### PR DESCRIPTION
**Pull request by zuhrinurhuda**

### CODE REVIEW

File: content/docs/lists-and-keys.md [hunk 1]
```diff
@@ -268,7 +268,7 @@ In the examples above we declared a separate `listItems` variable and included i
 ```js{3-6}
 function NumberList(props) {
   const numbers = props.numbers;
-  const listItems = numbers.map((number) =>
+  const listItems = numbers.map((number) => (
```
Comment (bvaughn) [inline content/docs/lists-and-keys.md]: This is valid:
```js
const numbers = [1, 2, 3, 4, 5];
const listItems = numbers.map((number) =>
  <li>{number}</li>
);
```
The parens are balanced.

The proposed change would be invalid though.

Maybe it helps if you add newlines:
```js
const numbers = [1, 2, 3, 4, 5];
const listItems = numbers.map(
  (number) => ( // New opening paren...
    <li>{number}</li>
  // But no closing paren
);
```

**Comment by bvaughn** [comment]
Thanks for the PR! Looks like it may be a mistake though. 😄

File: content/docs/lists-and-keys.md (hunk 1 above)
Comment (zuhrinurhuda) [reply content/docs/lists-and-keys.md]: Thanks for the explanation @bvaughn , sorry for my mistake..

Comment (bvaughn) [reply content/docs/lists-and-keys.md]: No worries!

**Pull request closed without merging**